go-auth.go

```diff
@@ -384,6 +386,20 @@ func setCache(authOpts map[string]string) {
 		}
 	}
 
+	if authJitterSec, ok := authOpts["auth_jitter_seconds"]; ok {
+		authSec, err := strconv.ParseInt(authJitterSec, 10, 64)
+		if err == nil {
+			authJitterSeconds = authSec
+		} else {
+			log.Warningf("couldn't parse authJitterSeconds (err: %s), defaulting to %d", err, authJitterSeconds)
```

Comment:
This is kind of odd because I would assume that passing a wrong value would default to the same as omitting the option, i.e. 0, not 3. What do you think?

Reply:
Either I'm missing something or I don't understand the first part of your comment.
For me (and my understanding of the code), the value used if option is omitted or in case of wrong value is 3.
Because in both case, we don't reach line 392 (`authJitterSeconds = authSec`) so we use using the default value set few line above.

I do agree that both omitted and wrong value must use the same default/fallback value.

I don't mind whether this default is 0 or 3. I've choose a default of jitter enabled because I think it only had benefit.

Reply:
Yeah, you're right that both are setting it to 3, I got thrown off by the docs which state that not passing a jitter would prevent the feature, i.e. the value should be 0, no 3. And if we're setting the default to 0, then I'd expect a parse error to default to 0 too.
Based on the principle of least surprise and if we wish to honor the docs, I think it makes sense to not have a jitter when not explicitly set by the user, and thus the default should be 0.

Reply:
Updated the default to 0.